src/convertToObject.js

```diff
@@ -6,7 +6,21 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const obj = {};
+
+  const stylesArray = stylesString
```

Comment:
The variable `stylesString` is used here but is not defined anywhere in the function. This will result in a ReferenceError. You need to ensure that `stylesString` is defined or passed as an argument to the function.